OMPython/__init__.py

```
@@ -447,3 +462,727 @@ def getClassNames(self, className=None, recursive=False, qualified=False, sort=F
+				self.qNamesList.append(q.name)
+		else:
+			pass
+		return self.qNamesList
+		
+	def checkAvailability(self, names, chkList, inputFlag = None):
+		isExist = True
+		if inputFlag is not None:
+			if names in chkList:
+				pass
+			else:
+				isExist = False
+			if(isExist == False):
+				print 'Error!!! ', names , ' does not exist.'
+				return False
+		
+		else:
+			nonExistingList = []
+			for n in names:
+				if n in chkList:
+					pass
+				else:
+					isExist = False
+					nonExistingList.append(n)
+
+		if(isExist == False):
+			print 'Error!!! ', nonExistingList , ' does not exist.'
+			return False
+		else:
+			return True
+	
+	def getQuantities(self, names = None):
+		str_ = ''
+		strList = []
+		if (names == None):
+			nameList = []
+			startList =[]
+			changableList = []
+			variabilityList = []
+			descriptionList = []
+			
+			for q in self.quantitiesList:
+				nameList.append(q.name)
+				startList.append(q.start)
+				changableList.append(q.changable)
+				variabilityList.append(q.variability)
+				descriptionList.append(q.description)
+
+			dic = {'Name' : nameList, 'Value' : startList, 'Changable' : changableList, 'Variability' : variabilityList, 'Description' : descriptionList}
+			frame = pd.DataFrame(dic)
+
+		else:
+			try:
+				checking = self.checkAvailability(names, self.qNamesList)
+				if(checking is False):
+					return None
+				
+				nameList = []
+				startList =[]
+				changableList = []
+				variabilityList = []
+				descriptionList = []
+				for n in names:
+					for q in self.quantitiesList:	
+						if n ==q.name:
+							nameList.append(q.name)
+							startList.append(q.start)
+							changableList.append(q.changable)
+							variabilityList.append(q.variability)
+							descriptionList.append(q.description)
+
+				dic = {'Name' : nameList, 'Value' : startList, 'Changable' : changableList, 'Variability' : variabilityList, 'Description' : descriptionList}	
+				frame = pd.DataFrame(dic)
+			except Exception as e:
+				return e
```

probably at least a `print e` should be added here, instead of silently swallowing any exception.